scenarioo-client/test/spec/shared/navigation/branchOrderBy.filter.spec.js

```diff
@@ -17,16 +17,75 @@
 
 'use strict';
 
+var CFG_LAST_BUILD_DATE_DESCENDING = { 'branchSelectionListOrder': 'last-build-date-descending' };
+var CFG_NAME_ASCENDING = { 'branchSelectionListOrder': 'name-ascending' };
+var CFG_NAME_DESCENDING = { 'branchSelectionListOrder': 'name-descending' };
+var CFG_PROP_NOT_SET = {};
+
+var DEFAULT_INPUT = [
+    {
+        alias: false,
+        branch: {
+            name: 'Ae'
```

Comment:
please use some more realistic test data like `release1.0`, `release1.1` and `release2.0` and maybe `master`.